fhd_core/beam_modeling/pyuvdata_beam_import.pro

```diff
@@ -0,0 +1,35 @@
+FUNCTION pyuvdata_beam_import, obs, antenna_str, pyuvdata_filepath,$
+    za_arr=za_arr, az_arr=az_arr, psf_image_dim=psf_image_dim, pix_use=pix_use
+
+lun = fxposit(pyuvdata_filepath, 0,/readonly)
+MRD_HREAD, lun, primary_header, /silent
+
+coordsys = strtrim(sxpar(primary_header,'coordsys'),2)
+CASE coordsys OF
```

Comment:
Can you put in comments which explain very briefly what the coordinate systems are?

Reply:
done